addons/iot_box_image/overwrite_before_init/etc/init_image.sh

```diff
@@ -206,12 +203,18 @@ chmod -R 644 /etc/logrotate.d/
 chown root:root /etc/logrotate.conf
 chmod 644 /etc/logrotate.conf
 
-echo "* * * * * rm /var/run/odoo/sessions/*" | crontab -
-
 update-rc.d -f hostapd remove
 update-rc.d -f nginx remove
 update-rc.d -f dnsmasq remove
 
+# Unfortunately labwc doesn't support hiding the mouse cursor, and the previous
+# solution (unclutter) doesn't work with Wayland. For now, this hack of setting
+# all the mouse cursor images to transparent is the only workaround.
+# We can remove this when this feature is merged: https://github.com/labwc/labwc/issues/2614
+cd /usr/share/icons/Adwaita/cursors
+for FILENAME in *; do rm $FILENAME; ln -s transparent $FILENAME; done
```

Comment:
```suggestion
for FILENAME in /usr/share/icons/Adwaita/cursors/*; do rm $FILENAME; ln -s transparent $FILENAME; done
```
Could be safer in case in the future the specified directory ceases to exist and we delete all of the files in the current working directory